platforms/macos/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+include(../shared/shared.cmake)
+findBoostCurl()
+buildSdk()
+
+add_subdirectory(../shared/tests)
```

Comment:
Will fix today. Sharing examples/tests code looks somewhat weird from cmake's point of view.